src/types/onyx/Paypal.ts

```diff
@@ -0,0 +1,27 @@
+import CONST from '../../CONST';
+import * as OnyxCommon from './OnyxCommon';
+
+type PaypalAccountData = {
+    username?: string;
+};
+
+type Paypal = {
+    /** This is always 'PayPal.me' */
+    title?: string;
+
+    /** The paypalMe address */
+    description?: string;
+
+    /** This is always 'payPalMe' */
+    methodID?: typeof CONST.PAYMENT_METHODS.PAYPAL;
+
+    /** This is always 'payPalMe' */
+    accountType?: typeof CONST.PAYMENT_METHODS.PAYPAL;
+
+    key?: string;
+    isDefault?: boolean;
+    pendingAction?: OnyxCommon.PendingAction;
+    accountData?: PaypalAccountData;
```

Comment:
```suggestion
    accountData: PaypalAccountData;
```

Looks like this is set here: https://github.com/Expensify/App/blob/405fcdf8850a62f6c02f96e34d71004144239a19/src/libs/actions/User.js#L507